### ISSUE TITLE
add relay instructions

### ISSUE THREAD

**Issue by omarcostahamido** [sauraen/HQStreamer2#3]
the readme only says:

"Relay
Open port 21568 for bidirectional TCP. Run hqs2relay in the background (Linux/Mac: ./hqs2relay > relay_cout.txt &)."

would be useful to have more details, like how to define the password, and how to launch it on windows.

Also, feature request: it would be nice to be able to choose the port. (And maybe this is too much to ask already, but it would be nice to have a step by step guide to host it, defining which are the requirements... can I host this in heroku, for example?)

Apologies in advance if my questions don't make sense or are way to simple.

**Comment by omarcostahamido** [comment]
Just tried it:
`$  ./hqs2relay > relay_cout.txt &`
and received:
```
[1] 3016
-bash: ./hqs2relay: Permission denied
[1]+  Exit 126                ./hqs2relay > relay_cout.txt
```

**Comment by sauraen** [comment]
I've added a bit more detail and Linux build instructions, plus an autorun system that put the server back up again if/when it crashes.

Not sure about the permission issue, may have to chmod the executable, though I think Git typically handles that correctly. Anyway it's expected that anyone running this on a real (Linux) server will build it from source, it only takes a minute and eliminates any potential compatibility issues.

I made the decision to use a fixed port for HQStreamer2, just to simplify the amount of entering info that non-tech-savvy users would have to do. Even watching interactions in a Discord of people trying to connect, people get confused between "server name" and "session name". If real-world issues arise with port conflicts, I'll certainly reconsider.
